client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -5,31 +5,56 @@
  */
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import cssSafeUrl from 'lib/css-safe-url';
 import Button from 'components/button';
+import safeImageUrl from 'lib/safe-image-url';
 
 export class ThemesBanner extends PureComponent {
 	static propTypes = {
-		title: PropTypes.string.isRequired,
-		description: PropTypes.string.isRequired,
+		title: PropTypes.oneOfType( [ PropTypes.string, PropTypes.object, PropTypes.array ] )
+			.isRequired,
+		description: PropTypes.oneOfType( [ PropTypes.string, PropTypes.object, PropTypes.array ] )
+			.isRequired,
```

Comment:
That seems a bit too permissive -- in what cases would `title` or `description` be an array or object?

Reply:
Good question! There are 3 things we can provide:

1) A String: `description={ "Hello World" }` 
2) An Object: `description={ <span>Hello <strong>World</strong></span> }`
3) An Array: `description={ translate( ... ) }`

The first is just a plain string, without translation. The second is a bare JSX node, which seems to be represented as an object internally. The third is the output of `translate()`.

I know it's too much, maybe we should just allow translated strings there? That seems more reallistic to me since the banner text would need to be translated anyway...

Reply:
Hmm, why is the result of `translate()` an array? That comes as a surprise to me...

Anyway, maybe `PropTypes.element` is the best fit here, since it allows any kind of React element; or `PropTypes.node`, which is even more permissive (allowing pretty much anything), but less verbose than `oneOfType( [ PropTypes.string, PropTypes.object, PropTypes.array ] )`.

(see [docs](https://www.npmjs.com/package/prop-types#usage) for available types)

Reply:
Oh, `PropTypes.element` should do the trick, thanks for the tip! :D